torchvision/transforms.py

```diff
@@ -701,10 +715,10 @@ def __call__(self, img):
             img (PIL.Image): Image to be flipped.
 
         Returns:
-            PIL.Image: Randomly cropped and scaled image.
+            PIL.Image: Randomly cropped and resize image.
         """
         i, j, h, w = self.get_params(img)
-        return scaled_crop(img, i, j, h, w, self.size, self.interpolation)
+        return resized_crop(img, i, j, h, w, self.size, self.interpolation)
```

Comment:
In this case, shouldn't we also rename `RandomScaledCrop` to `RandomResizedCrop`? This would fix https://github.com/pytorch/vision/issues/85 as well.

Reply:
👍 this is now done.